### ISSUE TITLE
Giant vendor directory chunk on run storybook caused by some issue described in a warning

### ISSUE THREAD

**Issue by seisenreich** [storybookjs/storybook#14248]
Hello there,
I'm getting a warning whenever I run `npm run storybook` resulting in a giant ~10 MiB vendor folder chunk and a warning.
Sure I'll provide you with all information probably missing here to find the actual problem.

1. cd into repository root
2. run `npm run storybook`
3. Look at the webpack output
4. See the warning

**Expected behavior**
I expected getting optimized chunks and no warnings / errors when trying to run storybook.

**Screenshots**
Output with colors - also uploaded as text file down below. 
![Output 1/2](https://user-images.githubusercontent.com/6771616/111331660-66db3300-8671-11eb-87cf-1f81815dd430.png)
![Output 2/2](https://user-images.githubusercontent.com/6771616/111331658-65aa0600-8671-11eb-9983-4b170eb1b82c.png)

**System**

Environment Info:

  System:
    OS: Linux 5.9 Manjaro Linux
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  Binaries:
    Node: 14.16.0 - ~/.nvm/versions/node/v14.16.0/bin/node
    Yarn: 1.22.10 - /usr/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v14.16.0/bin/npm
  Browsers:
    Firefox: 87.0b4
  npmPackages:
    @storybook/addon-a11y: ^6.1.20 => 6.1.20 
    @storybook/addon-actions: ^6.1.20 => 6.1.21 
    @storybook/addon-essentials: ^6.1.20 => 6.1.21 
    @storybook/addon-links: ^6.1.20 => 6.1.21 
    @storybook/react: ^6.1.20 => 6.1.21 

**Additional context**
Here some files I think could be helpful for you to help me
- [the typescript config](https://github.com/storybookjs/storybook/files/6150044/tsconfig.json.txt)
- [my package.json file](https://github.com/storybookjs/storybook/files/6150039/package.json.txt)
- [the output from the screenshot as text file](https://github.com/storybookjs/storybook/files/6150031/run-storybook-output.log)
- [.storybook/webpack.config.js.txt](https://github.com/storybookjs/storybook/files/6150075/webpack.config.js.txt)
- [.storybook/main.js](https://github.com/storybookjs/storybook/files/6150066/main.js.txt)
- [.storybook/preview.js](https://github.com/storybookjs/storybook/files/6150072/preview.js.txt)

**Comment by j3rem1e** [comment]
Probably dupe of #14160

**Comment by seisenreich** [comment]
Yup, looks like this is the same issue going on there.
When i remove a11y the warning is gone.

#14160  seems to be closed due to the fact, that the beta version of 6.2.x does not have this issue.
Is there a known solution for 6.1, too? I run storybook in a working environment used by developers, designers and marketing. I don't really want to use a beta version of this tool.

Otherwise I'll just probably wait until 6.2 is stable and hop on then.

Thank's for the quick response @j3rem1e

**Comment by shilman** [comment]
closing as dupe to #14160

6.2 should be available in the next few weeks

**Comment by seisenreich** [comment]
Thank you very much for taking the time - I appreciate it.
Then I'll simply wait for 6.2 :+1:
